PayloadPkg/Library/TrustyBootLib/TrustyBoot.h

```diff
@@ -42,6 +42,15 @@ typedef enum {
 
 #pragma pack (1)
 
+typedef struct {
+  UINT32 Eip;
+  UINT32 Eax;
+  UINT32 Ebx;
+  UINT32 Esi;
+  UINT32 Edi;
+  UINT32 Ecx;
+} CPU_BOOT_STATE;
```

Comment:
Moving this structure back into TrustyBoot.h since it cannot be used for pre-OS payload support (the registers used are different).